content/sql/concepts/window-functions/terms/lag/lag.md

```diff
@@ -0,0 +1,63 @@
+---
+Title: 'LAG()'
+Description: 'Accesses data from previous rows without the need for self join.'
+Subjects: 
+  - 'Data Science'
+Tags: 
+  - 'SQL Server'
+CatalogContent: 
+  - 'data-science-foundations'
+  - 'catalog/subject/data-science'
+---
+
+The **LAG()** window function facilitates access to previous rows based on the offset argument. It can be particularly useful when a comparison of a previous value is necessary without the use of a self join. There is a similarity to the `LEAD()` function with the difference being the accessible rows. `LEAD()` accesses subsequent rows while `LAG()` accesses previous rows.
+
+## Syntax
+
+```pseudo
+LAG (expression [, offset] [, default])  
+OVER ( [ partition_by ] order_by )
+```
+**expression** - the column value which will be referenced.
```

Comment:
Generally boldface is reserved for the first instance of the term and whenever we use  "Note: " to highlight a valuable property, caveat, etc. Otherwise, terms are either set off by using the backticks or italics. Please update the remaining parameters as I have above. Also remove the line breaks between the parameters.